test/integration/log_system.cc

```diff
@@ -1594,8 +1569,8 @@ TEST_F(LogSystemTest, LogResources)
   // Recorded models should exist
   EXPECT_GT(entryCount(recordPath), 2);
   EXPECT_TRUE(common::exists(common::joinPaths(recordPath, homeFake,
-      ".ignition", "fuel", "fuel.ignitionrobotics.org", "openrobotics",
-      "models", "x2 config 1")));
+      ".ignition", "fuel", "fuel.ignitionrobotics.org", "OpenRobotics",
```

Comment:
Looks like these lines and the following lines on 1607-1608 introduce a regression.  Moving them  back to lower case fixes the test case, is there any reason you uppercased these?

Reply:
Ohh I must have outdated fuel stuff locally. Changed back in 2c042bc